Source/WebCore/accessibility/isolatedtree/AXIsolatedObject.h

```diff
@@ -371,7 +372,9 @@ class AXIsolatedObject final : public AXCoreObject {
     AXIsolatedObject* highestEditableAncestor() final { return Accessibility::highestEditableAncestor(*this); }
     virtual std::optional<AccessibilityOrientation> explicitOrientation() const
     {
-        return propertyValue<std::optional<AccessibilityOrientation>>(AXProperty::ExplicitOrientation);
+        if (indexOfProperty(AXProperty::ExplicitOrientation) == notFound)
+            return std::nullopt;
+        return propertyValue<AccessibilityOrientation>(AXProperty::ExplicitOrientation);
```

Comment:
Doesn't this do two property vector scans? I wonder if we could use the result of the `indexOfProperty` call in this function to get the value if `index != notFound`. Could require a bit of code duplication though 🤔

Reply:
Good catch - I meant to replace this with optionalAttributeValue<T> like I did elsewhere but missed this one.